script/configs/mainnet/M2_mainnet_upgrade.config.json

```diff
@@ -0,0 +1,46 @@
+{
+  "chainInfo": {
+    "chainId": 1
+  },
+  "multisig_addresses": {
+    "communityMultisig": "0xFEA47018D632A77bA579846c840d5706705Dc598",
+    "executorMultisig": "0x369e6F597e22EaB55fFb173C6d9cD234BD699111",
+    "operationsMultisig": "0xBE1685C81aA44FF9FB319dD389addd9374383e90",
+    "pauserMultisig": "0x5050389572f2d220ad927CcbeA0D406831012390",
+    "timelock": "0xA6Db1A8C5a981d1536266D2a393c5F8dDb210EAF"
+  },
+  "strategies": {
+    "numStrategies": 0,
+    "MAX_PER_DEPOSIT": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+    "MAX_TOTAL_DEPOSITS": 115792089237316195423570985008687907853269984665640564039457584007913129639935,
+    "strategiesToDeploy": []
+  },
+  "strategyManager": {
+    "init_strategy_whitelister": "0xBE1685C81aA44FF9FB319dD389addd9374383e90",
+    "init_paused_status": 1
+  },
+  "delegationManager": {
+    "init_paused_status": 0,
+    "init_minWithdrawalDelayBlocks": 50400
+  },
+  "avsDirectory": {
+    "init_paused_status": 0
+  },
+  "slasher": {
+    "init_paused_status": 115792089237316195423570985008687907853269984665640564039457584007913129639935
+  },
+  "eigenPod": {
+    "MAX_RESTAKED_BALANCE_GWEI_PER_VALIDATOR": 32000000000000000000,
+    "GENESIS_TIME": 1606824000
+  },
+  "eigenPodManager": {
+    "init_paused_status": 0,
+    "deneb_fork_timestamp": "1710338135"
```

Comment:
Where are we getting this? Not that it's wrong, but I want a source to double check this.

Reply:
The Geth release has this https://github.com/ethereum/go-ethereum/releases/tag/v1.13.12

Reply:
https://github.com/ethereum/consensus-specs/blob/dev/specs/deneb/fork.md#configuration
https://beaconcha.in/epoch/269568
Fork epoch is `269568` and I converted to the timestamp

Reply:
Found the actual config in the specs here https://github.com/ethereum/consensus-specs/blob/68dceb7cda9f7745689fda007d712e6b128f94ac/configs/mainnet.yaml#L52